cmd/earthly/root_cmds.go

```diff
@@ -169,64 +170,76 @@ func (app *earthlyApp) rootCmds() []*cli.Command {
 		},
 		{
 			Name:        "account",
-			Usage:       "Create or manage an Earthly account *beta*",
+			Usage:       "Create or manage an Earthly account",
 			Subcommands: app.accountCmds(),
 		},
 		{
 			Name:        "debug",
 			Usage:       "Print debug information about an Earthfile",
-			Description: "Print debug information about an Earthfile",
+			Description: "Print debug information about an Earthfile.",
 			ArgsUsage:   "[<path>]",
 			Hidden:      true, // Dev purposes only.
 			Subcommands: app.debugCmds(),
 		},
 		{
-			Name:        "prune",
-			Usage:       "Prune Earthly build cache",
-			Description: "Prune Earthly build cache",
-			Action:      app.actionPrune,
+			Name:  "prune",
+			Usage: "Prune Earthly build cache",
+			Description: `Prune Earthly build cache in one of two forms.
+
+Standard Form:
+	Issues a prune command on the BuildKit daemon.
+Reset Form:
+	Restarts the BuildKit daemon and instructs it to complete delete the cache
+	directory on startup.
+		`,
+			Action: app.actionPrune,
 			Flags: []cli.Flag{
 				&cli.BoolFlag{
 					Name:        "all",
 					Aliases:     []string{"a"},
 					EnvVars:     []string{"EARTHLY_PRUNE_ALL"},
-					Usage:       "Prune all cache",
+					Usage:       "Prune all cache via BuildKit daemon",
 					Destination: &app.pruneAll,
 				},
 				&cli.BoolFlag{
-					Name:        "reset",
-					EnvVars:     []string{"EARTHLY_PRUNE_RESET"},
-					Usage:       "Reset cache entirely by wiping cache dir",
+					Name:    "reset",
+					EnvVars: []string{"EARTHLY_PRUNE_RESET"},
+					Usage: `Reset cache entirely by restarting BuildKit daemon and wiping cache dir.
+			This option is not available when using satellites.`,
 					Destination: &app.pruneReset,
 				},
+				&cli.DurationFlag{
+					Name: "age",
+					Usage: `Prune cache older than the specified duration passed in as a string.
+	Valid time units: ns, us, ms, s, n, h
```

Comment:
This seems overkill; I don't see a user-case for someone specifying a time based in nanoseconds.

I think `m, h, d` is good enough; (pretty sure it should be `m` and not `n` which looks like a typo).

```
Prune cache older than the specified duration passed in as a string; duration is specified with an integer value followed by a m, h, or d suffix which represents minutes, hours, or days respectively, e.g. 24h, or 1d
```
```